Makefile

```diff
@@ -11,19 +11,6 @@ current_dir := $(abspath $(patsubst %/,%,$(dir $(mkfile_path))))
 
 BUILDINGIMAGE=*
 
-# Docker PHP images build matrix ./build-php.sh (nts/zts) (PHP version) (Alpine version)
-build-nts: BUILDINGIMAGE=nts
```

Comment:
still referenced on line 2

Reply:
Yeah... need to fix the Makefile :zipper_mouth_face: